doc/contributing.asciidoc

```diff
@@ -696,24 +696,15 @@ qutebrowser release
 
 * Make sure there are no unstaged changes and the tests are green.
 * Make sure all issues with the related milestone are closed.
-* Run `x=... y=...` to set the respective shell variables.
-
-* Update changelog (remove *(unreleased)*).
-* Adjust `__version_info__` in `qutebrowser/__init__.py`.
-* Commit.
-
-* Create annotated git tag (`git tag -s "v1.$x.$y" -m "Release v1.$x.$y"`).
-* `git push origin`; `git push origin v1.$x.$y`.
```

Comment:
I don't think `bumpversion` takes care of the pushing?

Reply:
No, it doesn't, you are correct. It creates the bump commit and the related tag, but doesn't actually push it to remote.